docs/user_guide/modeling_extensions/uncertainty_propagation/index.rst

```diff
@@ -0,0 +1,48 @@
+=======================
```

Comment:
@ksbeattie @lbianchi-lbl Can you help @JanghoPark-LBL with an automated test for Sphynx building? We'll post more info in the PR thread in a few minutes.

Reply:
I fixed them and it works without errors.

Reply:
I confirmed this works on my computer.